bin/compile

```diff
@@ -10,17 +10,19 @@ env_dir="${3}"
 
 buildpack=$(cd "$(dirname $0)/.." && pwd)
 source "${buildpack}/lib/common.sh"
+snapshotBinBefore
+
 ensureInPath "jq-linux64" "${cache}/.jq/bin"
 
 # This is used by the buildpack stdlib for metrics
 BPLOG_PREFIX="buildpack.go"
 
-STDLIB_FILE=$(mktemp -t stdlib.XXXXX)
+STDLIB_DIR=$(mktemp -d -t stdlib.XXXXX)
 
 ### Load dependencies
 
-curl --silent --retry 5 --retry-max-time 15 'https://lang-common.s3.amazonaws.com/buildpack-stdlib/v8/stdlib.sh' > "$STDLIB_FILE"
-source "$STDLIB_FILE"
+ensureFile "stdlib.sh.v8" "${STDLIB_DIR}" "chmod a+x"
```

Comment:
this logs out to the console:

```
remote: -----> Fetching stdlib.sh.v8... done
```

I don't believe this information isn't helpful to the user. They don't know what a `stdlib.sh.v8` is, or why it would need to be part of their build process. I think we should only be logging something out if this download fails, which will be exceedingly rare.

Reply:
This is using the standard download and verify routines that everything else uses, so I'd really like to keep it and not have to continue special casing it.

Reply:
That's fair